lib/compiler-cranelift/Cargo.toml

```diff
@@ -14,21 +14,32 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
-wasmer-compiler = { path = "../compiler", version = "=4.3.5", features = ["translator", "compiler"], default-features = false }
-wasmer-types = { path = "../types", version = "=4.3.5", default-features = false, features = ["std"] }
-cranelift-entity = { version = "0.91.1", default-features = false }
-cranelift-codegen = { version = "0.91.1", default-features = false, features = ["x86", "arm64", "riscv64"] }
-cranelift-frontend = { version = "0.91.1", default-features = false }
+wasmer-compiler = { path = "../compiler", version = "=4.3.5", features = [
+	"translator",
+	"compiler",
+], default-features = false }
+wasmer-types = { path = "../types", version = "=4.3.5", default-features = false, features = [
+	"std",
+] }
+cranelift-entity = { version = "0.110.2", default-features = false }
+cranelift-codegen = { version = "0.110.2", default-features = false, features = [
+	"x86",
+	"arm64",
+	"riscv64",
+	"isle-in-source-tree"
+] }
+cranelift-frontend = { version = "0.110.2", default-features = false }
+itertools = "0.12.0"
 tracing = "0.1"
 hashbrown = { version = "0.11", optional = true }
 rayon = { version = "1.5", optional = true }
 more-asserts = "0.2"
-gimli = { version = "0.26", optional = true }
+gimli = { version = "0.28.0", optional = true }
```

Comment:
Nice. can we update gimli as well on the singlepass crate and make the version global for the Cargo workspace?

Reply:
Note: use the same gimli version as `cranelift-codegen` (that is: `0.28` (.1) )